app/views/middleware_server/_add_jdbc_driver.html.haml

```diff
@@ -0,0 +1,136 @@
+              &times;
+            %span.sr-only
+              Close
+          %h4.modal-title#mw_jdbc_modal_label
+            = _("Add JDBC Driver")
+        .modal-body
+          %input#jdbc_server_id{:type  => "hidden",
+                                :value => params[:id]}
+          .form-group
+            %label.col-md-4.control-label{:for => "enable_deployment_cb"}
+              = _("Select the file to deploy")
+            .col-md-8
+              = file_field("jdbc_driver", "file",
+                           :class             => "lifestyle",
+                           "data-buttonName"  => "btn-default",
+                           "ng-required"      => "true",
+                           "ng-model"         => "jdbcDriverModel.filePath",
+                           "require-file"     => "handle")
+
+          .form-group
+            %label.col-md-4.control-label{:for => "jdbc_driver_name_input"}
+              = _("Driver Name")
+            .col-md-8
+              %input.form-control#jdbc_driver_name_input{:type          => "text",
+                                                         "name"         => "jdbc_driver_name_input",
+                                                         "ng-disabled"  => "!jdbcDriverModel.filePath",
+                                                         "ng-required"  => "true",
+                                                         "ng-minlength" => "4",
+                                                         "ng-maxlength" => "100",
+                                                         "placeholder"  => _("Enter JDBC Driver name"),
+                                                         "ng-model"     => "jdbcDriverModel.driverName"}
+            .col-sm-2
+            .col-sm-10
+              %p.ng-error-messages{"ng-show" => "jdbcAddForm.jdbc_driver_name_input.$dirty && jdbcAddForm.jdbc_driver_name_input.$error.required"}
+                = _("Required")
+              %p.ng-error-messages{"ng-show" => "jdbcAddForm.jdbc_driver_name_input.$dirty && jdbcAddForm.jdbc_driver_name_input.$error.minlength"}
+                = _("Minimum length is 4")
+              %p.ng-error-messages{"ng-show" => "jdbcAddForm.jdbc_driver_name_input.$dirty && jdbcAddForm.jdbc_driver_name_input.$error.maxlength"}
+                = _("Maximum length is 100")
+
+
+          .form-group
+            %label.col-md-4.control-label{:for => "jdbc_module_name_input"}
+              = _("Module Name")
+            .col-md-8
+              %input.form-control#jdbc_module_name_input{:type           => "text",
+                                                          "name"         => "jdbc_module_name_input",
+                                                          "ng-disabled"  => "!jdbcDriverModel.filePath",
+                                                          "ng-required"  => "true",
+                                                          "ng-minlength" => "6",
```

Comment:
I am just curious, is there any reason for the min lengths? Is Hawkular server enforcing this?